components/molecules/BookNow.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { useColorScheme } from 'react-native'
+import { Text, type TextProps, XStack, YStack } from 'tamagui'
+
+import { PositiveButton } from '~/components/atoms/PositiveButton'
+import getColors from '~/constants/Colors'
+import { useAppFonts } from '~/hooks/useAppFonts'
+import useTranslation from '~/hooks/useTranslation'
+
```

Comment:
We need to check the conditions if the customer does not select the staff, the booking button will be blurred into gray to carefully select the time and the staff.